pkg/sac/scope_checker.go

```diff
@@ -14,7 +14,6 @@ import (
 //go:generate mockgen-wrapper
 type ScopeChecker interface {
 	SubScopeChecker(keys ...ScopeKey) ScopeChecker
-	TryAllowed(subScopeKeys ...ScopeKey) bool
```

Comment:
Shouldn't the function implementation (lines 58 to 66) be removed as well ?

Reply:
You're right it should be removed here not in https://github.com/stackrox/stackrox/pull/2778/files

Reply:
But it's used by other functions so let's keep it here for now

Reply:
ok, fine :)